### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by tituspijean**
Implement Tag's improvements:

- [x] #54 
- [x] #55 

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Review by Tagadda — APPROVED** [review]

**Review by Tagadda — COMMENTED** [review]
Package version bump ?

**Comment by tituspijean** [comment]
!testme

**Comment by tituspijean** [comment]
Sorry @Tagadda, but your fixes on the services do not work. I tried adding a new client, Apply config, start the VPN on the new client, and no, I still have to manually restart the WireGuard service. :/

File: scripts/upgrade [hunk 1]
```diff
@@ -107,6 +107,12 @@ if systemctl list-units --full -all | grep -Fq "wireguard_ui_conf.service"; then
 	ynh_secure_remove --file="/etc/systemd/system/wireguard_ui_conf.service"
 fi
 
+# Remove the service integration from an older version
+if ynh_exec_warn_less yunohost service status wireguard@wg0 >/dev/null
+then
+	yunohost service remove wireguard@wg0
+fi
+
```
[the remark this answers is not in the document]
Comment (Tagadda) [reply scripts/upgrade:115]: Should have been updated by https://github.com/YunoHost-Apps/wireguard_ynh/blob/aaad102c96c5144630c4a3d77df34316b5757557/scripts/upgrade#L194-L199, independently of this test I assume

File: scripts/_common.sh [hunk 2]
```diff
@@ -15,7 +15,7 @@ fi
 # dependencies used by the app
 pkg_dependencies="$pkg_headers wireguard-dkms wireguard"
 
-interface=$(ip route | awk '/default/ { print $5 }')
+interface=$(ip route | awk '/default/ { print $5 }' | head)
```
Comment (Tagadda) [inline scripts/_common.sh:18]:
```suggestion
interface=$(ip route | awk '/default/ { print $5 }' | head -n1)
```

**Review by Tagadda — CHANGES_REQUESTED** [review]

**Comment by tituspijean** [comment]
!testme

**Comment by tituspijean** [comment]
Thanks Tag!

**Pull request merged**